### PR TITLE
Publish Bitcoin docker images to use in local docker compose

### PR DESCRIPTION
**Pull request by nkuba**
Building docker images for Bitcoin local setup takes much more than 30 minutes, which is annoying to set up. With this PR we publish the images to GitHub Container Registry and use them in local docker compose.

### CODE REVIEW

File: bitcoin/docker-compose.yml [hunk 1]
```diff
@@ -3,13 +3,17 @@ version: "3"
 services:
   bitcoind:
     container_name: bitcoind
+    image: ghcr.io/keep-network/local-setup/bitcoind
+    # build: ./bitcoind
```
Comment (beaushinkle) [inline bitcoin/docker-compose.yml:7]: Can we get rid of the commented out `# build` lines?

**Comment by beaushinkle** [comment]
This is so cool!

File: bitcoin/docker-compose.yml (hunk 1 above)
Comment (nkuba) [reply bitcoin/docker-compose.yml:7]: I couldn't decide what's better. I left the commented lines for convenience if we want to test changes in any module configuration.

**Comment by beaushinkle** [comment]
merge conflicts!

**Comment by beaushinkle** [comment]
Just ran this locally. So cool to see it pull our images from the internet. It feels so official!

**Comment by nkuba** [comment]
@beaushinkle conflicts were solved, would you do the honors?

**Review by beaushinkle — APPROVED** [review]

**Pull request merged**